.github/workflows/labeler.yml

```diff
@@ -4,43 +4,29 @@ on:
     types: [opened]
   pull_request_target:
     types: [opened]
-env:
-  MY_GITHUB_TOKEN: ${{ secrets.APM_TECH_USER_TOKEN }}
+
+# 'issues: write' for https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#add-labels-to-an-issue
+permissions:
+  contents: read
+  issues: write
```

Comment:
I think 
```suggestion
  issues: write
  pull-requests: write
```

I think we also need `pull-requests: write` if this is adding labels to PRs aswell.

Reply:
The documentation says that one should be enough. This conf is taken from the nodejs agent but didn't receive any community PR in the meantime. I would like to try with this one and add the other if something is off.

Reply:
I was looking at https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

<img width="777" alt="image" src="https://github.com/elastic/apm-agent-python/assets/16325797/d8e7ff11-9a8a-4312-959b-671421e2a644">

Reply:
But feel free to try it out.